.github/workflows/check-secrets.yml

```
@@ -1,13 +1,10 @@
-name: (automatic) Check Secrets
-on: push
+name: Check Secrets
```

```suggestion
name: (automatic) Check Secrets
```

We usually do that to make it explicit which actions should be run manually.

Thanks for the PR

---

Is the definition of CI supposed to be automatic? Perhaps you want to add a flag to the workflows that are not triggered automatically? We don't have this term in any other repository in the company. Could you consider making a proposal about that in the [shared_workflows](https://github.com/revelotech/sre-iac/tree/main/.github/workflows)?